.travis.yml

```diff
@@ -46,11 +46,16 @@ install:
 
 script:
   - if [[ $BUILD_DOCS = true ]]; then
+      echo "buildign docs"
```

Comment:
Yeah.. just debug... trying stuff. I'll rbase clean later once I'm happy. I might do a few change to doctr first.